src/components/messenger/list/styles.scss

```diff
@@ -20,14 +20,17 @@ $side-padding: 16px;
   &__user-account-container {
     display: flex;
     align-items: center;
+    padding: 16px;
+    height: 32px;
+    gap: 8px;
+    background-color: theme.$color-primary-3;
   }
 
-  &__settings-menu-container {
-    display: flex;
-    padding-left: 16px;
-    width: 100%;
-    height: 64px;
-    background-color: theme.$color-primary-3;
+  &__user-name {
+    font-size: $font-size-small;
+    font-weight: 600;
+    line-height: 15px;
+    color: rgba(253, 253, 255, 0.93);
```

Comment:
Hmm I've directly added the rgba for the glass colour here, did we discuss how we are adding these colours, such as a colour library?